app/invocation/invocation_suggestion_card.tsx

```diff
@@ -161,15 +161,68 @@ const matchers: SuggestionMatcher[] = [
       level: SuggestionLevel.INFO,
       message: (
         <>
-          Adding the Bazel flag <span className="inline-code">--remote_download_minimal</span> or{" "}
-          <span className="inline-code">--remote_download_toplevel</span> can significantly improve performance for
-          builds with a large number of intermediate results.
+          Consider adding the Bazel flag <BazelFlag>--remote_download_minimal</BazelFlag> or{" "}
+          <BazelFlag>--remote_download_toplevel</BazelFlag>, which can significantly improve performance for builds with
+          a large number of intermediate results.
         </>
       ),
       reason: (
         <>
-          Shown because this build is cache-enabled and the effective flag{" "}
-          <span className="inline-code">--remote_download_outputs</span> is not set.
+          Shown because this build is cache-enabled and the flag{" "}
+          <span className="inline-code">--remote_download_outputs</span> is not explicitly set.
+        </>
+      ),
+    };
+  },
+  // Suggest using --incompatible_strict_action_env
+  ({ model }) => {
+    if (!capabilities.config.expandedSuggestionsEnabled) return null;
+
+    if (model.optionsMap.get("incompatible_strict_action_env") !== undefined) return null;
```

Comment:
@siggisim I wasn't sure whether we want to show this across the board, or only if remote cache is enabled. wdyt?

Reply:
I don't think we should suggest it until we can mark suggestions as "I don't want to do this". Most iOS projects shouldn't actually set this, and should instead filter `PATH` in `tools/bazel`, since `--incompatible_strict_action_env` adds `/usr/local/bin`, which picks up Homebrew on Intel Macs.

Reply:
Thanks for pointing this out -- removed from this PR.